### PR TITLE
Service implementation 

### PR DESCRIPTION
**Pull request by lkenigsb**
Screen Shot 2023-05-28 at 1 32 16 PM
Screen Shot 2023-05-28 at 1 32 28 PM

### CODE REVIEW

File: src/main/java/kenigsberg/transit/TransitService.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface TransitService {
+    @GET("api/siri/stop-monitoring.json?key=8824cb1a-61a5-4a89-827c-" +
+            "d66114bb7f12&OperatorRef=MTA&LineRef=MTA%SIM33C")
+    Observable<StopMonitor> getTransitInfo(@Query("MonitoringRef") String monitoringRefNum);
+
+    //Ref Num: 203611 (sometimes limited info)
+    //Ref Num: 404993
```
Comment (andrewoid) [inline src/main/java/kenigsberg/transit/TransitService.java:14]: How can I find out what ref numbers work?

Comment (lkenigsb) [reply src/main/java/kenigsberg/transit/TransitService.java:14]: Do you mean a list of Ref Numbers? That I had to google...

Comment (andrewoid) [reply src/main/java/kenigsberg/transit/TransitService.java:14]: Where is that listed?

File: src/main/java/kenigsberg/transit/TransitFrame.java [hunk 2]
```diff
@@ -55,10 +58,23 @@ public TransitFrame() {
         detailStopPanel.add(limitedInfoLabel);
 
 
-        controller = new TransitController(service, limitedInfoLabel, detailStopPanel);
+        DefaultListModel listModel = new DefaultListModel();
+
+        JList list = new JList(listModel);
```
Comment (andrewoid) [inline src/main/java/kenigsberg/transit/TransitFrame.java]: JTable, not JList

Comment (andrewoid) [reply src/main/java/kenigsberg/transit/TransitFrame.java]: I may have misspoke when we talked about this https://docs.oracle.com/javase/tutorial/uiswing/components/table.html

File: src/main/java/kenigsberg/transit/TransitFrame.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            .addConverterFactory(GsonConverterFactory.create())
+            .addCallAdapterFactory(RxJava3CallAdapterFactory.create())
+            .build();
+
+
+    private TransitService service = retrofit.create(TransitService.class);
+
+    private JLabel limitedInfoLabel;
+    private TransitController controller;
+    String[] listContent = new String[20];
```
Comment (andrewoid) [inline src/main/java/kenigsberg/transit/TransitFrame.java]: Not used anymore?

**Review by andrewoid — COMMENTED** [review]
Please include a screenshot in your PR.

**Review by andrewoid — APPROVED** [review]
Try adding Dagger now in a different PR.

**Pull request merged**